nengo/tests/conftest.py

```diff
@@ -232,6 +233,11 @@ def pytest_runtest_setup(item):
         if skip:
             pytest.skip(" and ".join(skipreasons))
 
+    if 'Simulator' in item.fixturenames:
+        for test, reason in _Simulator.unsupported:
+            if re.match(test, item.name):
```

Comment:
Looking at nengo/nengo_ocl#65, I think we should definitely switch to `item.nodeid` before merging this. Otherwise, we might be skipping a lot of tests we don't mean to (there are a number of tests in different files that have the same name, so this is a real issue). We don't use tests in classes at all, AFAIK, so I don't think that we need to be worried about that. If we do decide to use tests in classes down the road, then we can always write a function that takes `item.nodeid` and formats it more nicely. But for now I think `item.nodeid` is fine.